src/services/getAppInfo.js

```diff
@@ -1,20 +1,22 @@
 'use strict';
 
 const express = require('express');
+const config = require('../config');
+const pkgJSON = require('../../package.json');
+const gitData = require('../services/getGitData');
 
 const app = express();
 const env = app.get('env');
 
-const pkgJSON = require('../../package.json');
-
-function getAppInfo() {
+async function getAppInfo() {
+  const gitInfo = await gitData();
   return new Promise((resolve) => {
     resolve({
       title: pkgJSON.name,
       environment: env,
       version: pkgJSON.version,
-      commit: 'TODO!',
-      build: 'TODO!',
+      commit: config.herokuSlugCommit || gitInfo.long,
+      build: config.herokuSlugDescription || gitInfo.short,
```

Comment:
We may as well drop the build. It doesn't really add value. The build should be in the version number... which should be incremented by an automated build process (can you say, "out of scope"?).